src/common/storage/local-storage-manager.js

```diff
@@ -0,0 +1,114 @@
+// @flow
+import StorageWrapper from './storage-wrapper';
+import {EventManager, getLogger} from '@playkit-js/playkit-js';
+import {BaseStorageManager} from './base-storage-manager';
+
+export default class LocalStorageManager extends BaseStorageManager {
+  static StorageKeys: {[key: string]: string} = {
+    MUTED: 'muted',
+    VOLUME: 'volume',
+    AUDIO_LANG: 'audioLanguage',
+    TEXT_LANG: 'textLanguage',
+    CAPTIONS_DISPLAY: 'captionsDisplay',
+    TEXT_STYLE: 'textStyle'
+  };
+
+  static _logger: any = getLogger('StorageManager');
+
+  static getStorageObject() {
+    return localStorage;
+  }
+
+  /**
+   * @static
+   * @private
+   * @returns {boolean} - Whether local storage is implemented in the current browser.
+   */
+  static isLocalStorageAvailable(): boolean {
```

Comment:
i think we can just rename it to isStorageAvailable and put it on the interface, no ?
if we keep adding managers and every one of them will have isXStorageAvailable it will be redundant because we know what manager we are invoking this on

Reply:
basically we will not have more storage managers that use this API because there are only 2- the session storage and local storage. other storages are being used in different ways or using different APIs. however, i do agree that it does make sense to move it to the base as well since both of them doing the same. will do it.